server/services/store/storetests/blocks.go

```diff
@@ -966,12 +966,12 @@ func testGetBlockMetadata(t *testing.T, store store.Store) {
 	})
 
 	t.Run("get block history after updateAt", func(t *testing.T) {
-		rBlocks, err2 := store.GetBlocksWithType(boardID, "test")
```

Comment:
These tests were relying on the order of the full-scan to work properly, which now changes with the index. They've been updated to directly access the specific block needed